sympy/tensor/indexed.py

```diff
@@ -1,115 +1,10 @@
-"""Module that defines indexed objects
```

Comment:
My rationale was that this content is an intro to both ``indexed`` as well as ``indexed_sums`` (and formerly ``index_methods``). So I thought it was more appropriate for it not to be in the ``indexed`` module. It also makes the documentation outline cleaner:
* Indexed Objects
  - Basic Examples
  - Reference \<this is where module docstrings are imported\>
    - Indexed Objects
    - Implicit Sums (Einstein Notation)
* Tensor

as opposed to the current:

* Indexed Objects
  - Examples
* Methods
* Tensor